waddrmgr/manager_test.go

```diff
@@ -1255,6 +1281,26 @@ func testLookupAccount(tc *testContext) bool {
 	if !checkManagerError(tc.t, testName, err, wantErrCode) {
 		return false
 	}
+
+	// Test address account lookup
+	var expectedAccount uint32
+	addressHash := hexToBytes("2ef94abb9ee8f785d087c3ec8d6ee467e92d0d0a")
```

Comment:
Since the seed is part of the global test binary, I'd like to see all of these hardcoded expected addresses to either also be globals, or passed around as part of the test context.